hera_cal/abscal_funcs.py

```diff
@@ -737,8 +740,8 @@ def global_phase_slope_logcal(model, data, antpos, wgts=None, refant=None, verbo
         delta_phi[binary_flgs] *= np.nan
         avg_wgts[rk][np.isinf(delta_phi)+np.isnan(delta_phi)] = 0.0
         delta_phi[np.isinf(delta_phi)+np.isnan(delta_phi)] *= np.nan
-        ls_data[eqn_str] = np.nanmedian(delta_phi, axis=1, keepdims=True)
-        ls_wgts[eqn_str] = np.sum(avg_wgts[rk], axis=1, keepdims=True)
+        ls_data[eqn_str] = np.nanmedian(delta_phi[:, edge_cut:(len(delta_phi)-edge_cut)], axis=1, keepdims=True)
```

Comment:
are you sure this is right?  I think `len(delta_phi)` will give you the `Ntimes`, which is not what you want. what's the problem with doing 
```
if edge_cut > 0:
    ls_data[eqn_str] = np.nanmedian(delta_phi[:, edge_cut:-edge_cut], axis=1)
else:
    ls_data[eqn_str] = np.nanmedian(delta_phi, axis=1)
```
?

Reply:
Fixed. Good catch.